src/pages/EnablePayments/OnfidoPrivacy.js

```diff
@@ -53,28 +53,16 @@ class OnfidoPrivacy extends React.Component {
     }
 
     render() {
-        let onfidoError = lodashGet(this.props, 'walletOnfidoData.error') || '';
-        if (!onfidoError) {
-            const onfidoFixableErrors = lodashGet(this.props, 'userWallet.onfidoFixableErrors', []);
-            if (!_.isEmpty(onfidoFixableErrors)) {
-                const supportedErrorKeys = ['originalDocumentNeeded', 'documentNeedsBetterQuality', 'imageNeedsBetterQuality', 'selfieIssue', 'selfieNotMatching', 'selfieNotLive'];
-                const translatedFixableErrors = _.filter(_.map(onfidoFixableErrors, (errorKey) => {
-                    if (_.contains(supportedErrorKeys, errorKey)) {
-                        return Localize.translateLocal(`onfidoStep.${errorKey}`);
-                    }
-                    return null;
-                }));
-                if (!_.isEmpty(translatedFixableErrors)) {
-                    onfidoError = translatedFixableErrors.join(' ');
-                }
-            }
-        }
+        const errors = lodashGet(this.props, 'walletOnfidoData.errors', {});
+        let onfidoError = _.isEmpty(errors) ? '' : _.last(_.values(errors));
+        const onfidoFixableErrors = lodashGet(this.props, 'walletOnfidoData.fixableErrors', []);
+        onfidoError += !_.isEmpty(onfidoFixableErrors) ? `<br/> ${onfidoFixableErrors.join('<br>')}` : '';
```

Comment:
I think we should use `\n` because afaik <br> or <br/> only works on web and it's ignored on native platforms